### PR TITLE
Travis improvements

### PR DESCRIPTION
**Pull request by awlodarkiewicz**
New test time:  1 min 55 sec

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,59 +53,70 @@ after_script:
...
+    - in_docker ./vendor/bin/phpcs -s --standard=PHPCompatibility app tests"
+
+    # PHP_CodeSniffer: check PSR2 standard
+    - in_docker ./vendor/bin/phpcs -s --standard=PSR2 app tests"
+
+    # SonarQube
+    - time sonar-scanner
+
+  - stage: build and install
+    if: type = pull_request
```
Comment (c3zi) [inline .travis.yml]: It should be run only when we want to merge into master. Shouldn't it?

```if: type = pull_request AND branch=master```

File: .travis.yml [hunk 2]
```diff
@@ -52,59 +53,70 @@ after_script:
...
+    - in_docker /bin/bash scripts/pre-build.sh"
+
+    # Run build script
+    - in_docker /bin/bash scripts/build.sh"
+
+    # PHP unit tests
+    - in_docker ./vendor/bin/phpunit --testsuite Console,Http,Unit"
+
+  - stage: code quality
+    if: type = pull_request
```
Comment (c3zi) [inline .travis.yml]: Code quality stage should be run for every commit ...

**Review by c3zi — DISMISSED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ services:
 
 env:
   global:
-    - DOCKER_IMAGE=adshares/adserver:devops
+    - DOCKER_IMAGE=awlodarkiewicz/devops:adserver
```
Comment (teghnet) [inline .travis.yml:12]: What is wrong with `adshares/adserver:devops`?
We can create `adshares/devops:adserver`.

File: .travis.yml [hunk 4]
```diff
@@ -18,20 +18,20 @@ env:
     - APP_DEBUG=true
 
 stages:
-  - test
-#  - build and install
+  - code test and quality
```
Comment (teghnet) [inline .travis.yml:21]:
```suggestion
  - code quality and testing
```

File: .travis.yml [hunk 5]
```diff
@@ -52,59 +52,61 @@ after_script:
...
-
-        - in_docker /bin/bash scripts/pre-build.sh"
-
-        # Run build script
-        - in_docker /bin/bash scripts/build.sh"
-        - in_docker /bin/bash scripts/pre-install.sh"
-
-        - docker ps -a
-        - in_docker /bin/bash scripts/install.sh"
+  - stage: code test and quality
```
Comment (teghnet) [inline .travis.yml:55]:
```suggestion
  - stage: code quality and testing
```

**Review by teghnet — CHANGES_REQUESTED** [review]

**Review by teghnet — APPROVED** [review]

**Pull request merged**